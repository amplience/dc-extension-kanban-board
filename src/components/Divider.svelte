<script>
  export let vertical: string = '';
</script>

<style lang="scss">
  .divider {
    &.vertical {
      border-right: 1px solid #e5e5e5;
      line-height: 1;
      margin: 0 16px;
    }

    &.horizontal {
      border-bottom: 1px solid #e5e5e5;
      width: 100%;
    }
  }
</style>

<div class="divider {vertical ? 'vertical' : 'horizontal'}" />
