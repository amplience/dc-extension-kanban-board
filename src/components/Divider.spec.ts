import { render } from '@testing-library/svelte';
import Divider from './Divider.svelte';

describe('divider component', () => {
  it('should render a divider component', () => {
    const { container } = render(Divider, {
      // no-op.
    });

    expect(container.firstChild).toMatchSnapshot();
  });
});
