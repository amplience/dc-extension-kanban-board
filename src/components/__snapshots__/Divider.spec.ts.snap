// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`divider component should render a divider component 1`] = `
<div>
  <div
    class="divider horizontal svelte-1rcmn0v"
  />
</div>
`;
